reporter/src/main/kotlin/reporters/freemarker/FreemarkerTemplateProcessor.kt

```diff
@@ -113,16 +115,17 @@ class FreemarkerTemplateProcessor(
         }
 
         // Keep this in sync with "resources/templates/freemarker_implicit.ftl".
-        val dataModel = mapOf(
-            "projects" to projects,
-            "packages" to packages,
-            "ortResult" to input.ortResult,
-            "licenseTextProvider" to input.licenseTextProvider,
-            "LicenseView" to LicenseView,
-            "helper" to TemplateHelper(input),
-            "projectsAsPackages" to projectsAsPackages,
-            "vulnerabilityReference" to VulnerabilityReference
-        )
+        val dataModel =
+            mapOf(
```

Comment:
Nit: Formatting should stay at `val dataModel = mapOf(` and following lines be de-intended (to keep the diff small for review convenience).